index.bs

```diff
@@ -652,6 +653,11 @@ This method accepts two arguments:
     ::  This argument is a {{CredentialCreationOptions}} object whose
         <code>|options|.{{CredentialCreationOptions/publicKey}}</code> member contains a {{MakePublicKeyCredentialOptions}}
         object specifying the desired attributes of the to-be-created [=public key credential=].
+
+    :   <dfn>sameOriginWithAncestors</dfn>
+    ::  This argument is a boolean which is true iff the caller's [=environment settings object=] is
```

Comment:
Do we use "iff" anywhere else in the document? Is it normally well understood for W3 specs?

Reply:
@mikewest uses it in credman.  we can spell it out i spose.